apps/frontend/src/app/5_pages/AavePage/AavePage.utils.tsx

```diff
@@ -157,3 +158,6 @@ export const sortRowsByOrderOptions = (
       : -direction,
   );
 };
+
+export const formatFloorNumber = (num: Decimal, precision = 2) =>
+  Math.floor(num.toNumber() * 10 ** precision) / 10 ** precision;
```

Comment:
We already have similar functions, such as `formatValue` or others, in the `apps/frontend/src/utils/math.ts` file. Please double-check and use them instead of creating new ones.